### ISSUE TITLE
训练分类时卡死，没有任何提示

### ISSUE THREAD

**Issue by Atiuiee** [PaddlePaddle/PaddleOCR#4663]
/home/ps/ininin/yqd/inProject/PaddleOCR/venv/bin/python /media/ps/data/software/pycharm/pycharm-community-2021.2.3/plugins/python-ce/helpers/pydev/pydevd.py --multiproc --qt-support=auto --client 127.0.0.1 --port 42329 --file /home/ps/ininin/yqd/inProject/PaddleOCR/PaddleOCR-release-2.3/tools/train.py
Connected to pydev debugger (build 212.5457.59)
/home/ps/ininin/yqd/inProject/PaddleOCR/venv/lib/python3.8/site-packages/paddle/fluid/layers/utils.py:26: DeprecationWarning: `np.int` is a deprecated alias for the builtin `int`. To silence this warning, use `int` by itself. Doing this will not modify any behavior and is safe. When replacing `np.int`, you may wish to use e.g. `np.int64` or `np.int32` to specify the precision. If you wish to review your current use, check the release note link for additional information.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  def convert_to_list(value, n, name, dtype=np.int):
/home/ps/ininin/yqd/inProject/PaddleOCR/venv/lib/python3.8/site-packages/skimage/morphology/_skeletonize.py:241: DeprecationWarning: `np.bool` is a deprecated alias for the builtin `bool`. To silence this warning, use `bool` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.bool_` here.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  0, 1, 1, 0, 0, 1, 0, 0, 0], dtype=np.bool)
/home/ps/ininin/yqd/inProject/PaddleOCR/venv/lib/python3.8/site-packages/skimage/morphology/_skeletonize.py:256: DeprecationWarning: `np.bool` is a deprecated alias for the builtin `bool`. To silence this warning, use `bool` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.bool_` here.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  0, 0, 0, 0, 0, 0, 0, 0, 0], dtype=np.bool)
[2021/11/16 15:54:15] root INFO: Architecture : 
[2021/11/16 15:54:15] root INFO:     Backbone : 
[2021/11/16 15:54:15] root INFO:         model_name : small
[2021/11/16 15:54:15] root INFO:         name : MobileNetV3
[2021/11/16 15:54:15] root INFO:         scale : 0.35
[2021/11/16 15:54:15] root INFO:     Head : 
[2021/11/16 15:54:15] root INFO:         class_dim : 4
[2021/11/16 15:54:15] root INFO:         name : ClsHead
[2021/11/16 15:54:15] root INFO:     Neck : None
[2021/11/16 15:54:15] root INFO:     Transform : None
[2021/11/16 15:54:15] root INFO:     algorithm : CLS
[2021/11/16 15:54:15] root INFO:     model_type : cls
[2021/11/16 15:54:15] root INFO: Eval : 
[2021/11/16 15:54:15] root INFO:     dataset : 
[2021/11/16 15:54:15] root INFO:         data_dir : /home/ps/ininin/yqd/inProject/PaddleOCR/PaddleOCR-release-2.3/train_data/cls/
[2021/11/16 15:54:15] root INFO:         label_file_list : ['/home/ps/ininin/yqd/inProject/PaddleOCR/PaddleOCR-release-2.3/train_data/cls/test.txt']
[2021/11/16 15:54:15] root INFO:         name : SimpleDataSet
[2021/11/16 15:54:15] root INFO:         transforms : 
[2021/11/16 15:54:15] root INFO:             DecodeImage : 
[2021/11/16 15:54:15] root INFO:                 channel_first : False
[2021/11/16 15:54:15] root INFO:                 img_mode : BGR
[2021/11/16 15:54:15] root INFO:             ClsLabelEncode : None
[2021/11/16 15:54:15] root INFO:             ClsResizeImg : 
[2021/11/16 15:54:15] root INFO:                 image_shape : [3, 48, 192]
[2021/11/16 15:54:15] root INFO:             KeepKeys : 
[2021/11/16 15:54:15] root INFO:                 keep_keys : ['image', 'label']
[2021/11/16 15:54:15] root INFO:     loader : 
[2021/11/16 15:54:15] root INFO:         batch_size_per_card : 16
[2021/11/16 15:54:15] root INFO:         drop_last : False
[2021/11/16 15:54:15] root INFO:         num_workers : 4
[2021/11/16 15:54:15] root INFO:         shuffle : False
[2021/11/16 15:54:15] root INFO: Global : 
[2021/11/16 15:54:15] root INFO:     cal_metric_during_train : True
[2021/11/16 15:54:15] root INFO:     checkpoints : None
[2021/11/16 15:54:15] root INFO:     debug : False
[2021/11/16 15:54:15] root INFO:     distributed : False
[2021/11/16 15:54:15] root INFO:     epoch_num : 100
[2021/11/16 15:54:15] root INFO:     eval_batch_step : [0, 1000]
[2021/11/16 15:54:15] root INFO:     infer_img : doc/imgs_words_en/word_10.png
[2021/11/16 15:54:15] root INFO:     label_list : ['ch', 'en', 'japan', 'korean']
[2021/11/16 15:54:15] root INFO:     log_smooth_window : 20
[2021/11/16 15:54:15] root INFO:     pretrained_model : None
[2021/11/16 15:54:15] root INFO:     print_batch_step : 10
[2021/11/16 15:54:15] root INFO:     save_epoch_step : 3
[2021/11/16 15:54:15] root INFO:     save_inference_dir : None
[2021/11/16 15:54:15] root INFO:     save_model_dir : ./output/cls/mv3/
[2021/11/16 15:54:15] root INFO:     use_gpu : True
[2021/11/16 15:54:15] root INFO:     use_visualdl : False
[2021/11/16 15:54:15] root INFO: Loss : 
[2021/11/16 15:54:15] root INFO:     name : ClsLoss
[2021/11/16 15:54:15] root INFO: Metric : 
[2021/11/16 15:54:15] root INFO:     main_indicator : acc
[2021/11/16 15:54:15] root INFO:     name : ClsMetric
[2021/11/16 15:54:15] root INFO: Optimizer : 
[2021/11/16 15:54:15] root INFO:     beta1 : 0.9
[2021/11/16 15:54:15] root INFO:     beta2 : 0.999
[2021/11/16 15:54:15] root INFO:     lr : 
[2021/11/16 15:54:15] root INFO:         learning_rate : 0.001
[2021/11/16 15:54:15] root INFO:         name : Cosine
[2021/11/16 15:54:15] root INFO:     name : Adam
[2021/11/16 15:54:15] root INFO:     regularizer : 
[2021/11/16 15:54:15] root INFO:         factor : 0
[2021/11/16 15:54:15] root INFO:         name : L2
[2021/11/16 15:54:15] root INFO: PostProcess : 
[2021/11/16 15:54:15] root INFO:     name : ClsPostProcess
[2021/11/16 15:54:15] root INFO: Train : 
[2021/11/16 15:54:15] root INFO:     dataset : 
[2021/11/16 15:54:15] root INFO:         data_dir : /home/ps/ininin/yqd/inProject/PaddleOCR/PaddleOCR-release-2.3/train_data/cls/
[2021/11/16 15:54:15] root INFO:         label_file_list : ['/home/ps/ininin/yqd/inProject/PaddleOCR/PaddleOCR-release-2.3/train_data/cls/train.txt']
[2021/11/16 15:54:15] root INFO:         name : SimpleDataSet
[2021/11/16 15:54:15] root INFO:         transforms : 
[2021/11/16 15:54:15] root INFO:             DecodeImage : 
[2021/11/16 15:54:15] root INFO:                 channel_first : False
[2021/11/16 15:54:15] root INFO:                 img_mode : BGR
[2021/11/16 15:54:15] root INFO:             ClsLabelEncode : None
[2021/11/16 15:54:15] root INFO:             RecAug : 
[2021/11/16 15:54:15] root INFO:                 use_tia : False
[2021/11/16 15:54:15] root INFO:             RandAugment : None
[2021/11/16 15:54:15] root INFO:             ClsResizeImg : 
[2021/11/16 15:54:15] root INFO:                 image_shape : [3, 48, 192]
[2021/11/16 15:54:15] root INFO:             KeepKeys : 
[2021/11/16 15:54:15] root INFO:                 keep_keys : ['image', 'label']
[2021/11/16 15:54:15] root INFO:     loader : 
[2021/11/16 15:54:15] root INFO:         batch_size_per_card : 16
[2021/11/16 15:54:15] root INFO:         drop_last : True
[2021/11/16 15:54:15] root INFO:         num_workers : 8
[2021/11/16 15:54:15] root INFO:         shuffle : True
[2021/11/16 15:54:15] root INFO: train with paddle 2.0.0 and device CUDAPlace(0)
[2021/11/16 15:54:15] root INFO: Initialize indexs of datasets:['/home/ps/ininin/yqd/inProject/PaddleOCR/PaddleOCR-release-2.3/train_data/cls/train.txt']
[2021/11/16 15:54:15] root INFO: Initialize indexs of datasets:['/home/ps/ininin/yqd/inProject/PaddleOCR/PaddleOCR-release-2.3/train_data/cls/test.txt']
W1116 15:55:42.623760 36207 device_context.cc:362] Please NOTE: device: 0, GPU Compute Capability: 8.6, Driver API Version: 11.2, Runtime API Version: 10.2
W1116 15:55:42.626085 36207 device_context.cc:372] device: 0, cuDNN Version: 7.6.
调试时发现卡在这一行
![945ff08af583ad43974137cf541041d](https://user-images.githubusercontent.com/31364852/141944305-6f1e24f0-f7b6-46d5-ba2d-aee31fabe329.png)

**Comment by Atiuiee** [comment]
已解决，安装了两个CUDA版本11.2、10.0，卸了10.2就可以了
